### DISCUSSION TITLE
2022-01-31 Core team update

### DISCUSSION THREAD

**Discussion by mm-near** [near/community#202] [Core]
# TL;DR

* debugging issues with OOM related to the new release
* more storage / rocksdb analysis & planned improvements
* preparing designs on how to stress-test nearcore (network, storage and chain code)

# Done

@mzhangmzz - Debug and test fix for testnet OOM and syncing issues

@abacabadabacaba - new storage prototype and RocksDB research
@EgorKulikov - design doc for configuration updates
@EgorKulikov - Designing tools for saving data for regression tests / simulations

@Longarithm - Fixing important bug in our caching code
@Longarithm - prototype of chunk nodes cache (that should safe x1.3 on Aurora)
@Longarithm - Analyzing undercharging in transactions

@mm-near - debugging OOM issues related to the recent release
@mm-near - designing how to stress test the system
